cnf-certification-test/results/html/results.html

```diff
@@ -777,7 +778,7 @@
           fillResults(json.claim.results, classification, '#mandatory-telco-table', '#optional-telco-table', "telco")
           fillResults(json.claim.results, classification, '#mandatory-non-telco-table', '#optional-non-telco-table', "nontelco")
           fillResults(json.claim.results, classification, '#mandatory-extended-table', '#optional-extended-table', "extended")
-
+          fillFeedback(feedback)         
```

Comment:
Shouldn't the `cnf-certification-test/results/html/results-embed.html` file be updated as well?

Reply:
yah we have it also as upload its ok, we have defult value that nothing in all